README.md

```diff
@@ -22,6 +22,11 @@ LPH.
 
 </div>
  
+
+# How to contribute : 
+
+  ℹ️ [contributing documentation](CONTRIBUTING.md)
+
```

Comment:
add a ">" before the emoji
but why did you used that emoji ?

Reply:
The emoji seem adapted for me.